### PR TITLE
Feature/segment improvements

### PR DESCRIPTION
**Pull request by alexey-rise**
## Description
Switched to remote Segment scripts (with permissions), added version to tracking, added staging vs production key, renamed icon for consistency, removed New Tab tracking, made Staging configuration default (so that if someone checks out repo from GitHub they don't start generating production stats on their development machine).

## Motivation and Context
As discussed we only need to track Install/Update/Startup, we also need to support staging vs production environment

## How Has This Been Tested?
Manually.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
N/A

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+analytics.js
```
Comment (tejohnso) [inline .eslintignore]: Can you add this into package.json ignorePatterns instead of adding a new file?
Or take that out and do it all this way, but let's not use two ways to ignore files for linting.

File: src/background.js [hunk 2]
```diff
@@ -1,9 +1,7 @@
 chrome.runtime.onInstalled.addListener(details=>{
-  analytics.track("Install", {reason: details.reason});
+  analytics.track("Install", {reason: details.reason, version: chrome.app.getDetails().version});
 });
 
 chrome.runtime.onStartup.addListener(()=>{
-  analytics.track("Startup", {test: "test"});
-});
-
-chrome.browserAction.onClicked.addListener(()=>chrome.tabs.create({}));
```
Comment (tejohnso) [inline src/background.js:9]: Why is this removed?

**Review by tejohnso — CHANGES_REQUESTED** [review]

**Comment by alexey-rise** [comment]
@tejohnso please see the last commit.

**Review by tejohnso — APPROVED** [review]
